### ISSUE TITLE
Accordion show/hide all button doesn't announce current state of accordions (DAC usability)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2253]
DAC suggest adding `aria-expanded` to the 'show all' and 'hide all' buttons.

(Page 86-87 of June 2022 DAC DS report...)

---

## GOV Accordions (Usability)

Accordions do not follow the GOV Design System specifications.

### Reference:
Usability & GOV.UK Design System requirement

### Issue ID: DAC_Usability_06
URL: https://ons-design-system.netlify.app/components/accordion/examples/accordion/
Page title: Example
Journey: Journey 1, Step 1, b

### Screenshot:
Accordion pages provide a ‘Show all’ button. The button can expand all of the accordions on the page. There is no aria-expanded attribute for the show all button; however, the accordion links have an aria-expanded attribute. This may cause some confusion to screen reader users.

### Current code ref(s):
#accordion-1 > div.ons-collapsible__heading.ons-js-collapsible-heading 
```
<button type="button" class="ons-btn ons-js-collapsible-all ons-u-mb-s ons-btn--secondary ons-btn--small" data-close-all="Hide all" data-group="accordion" data-ga-label="Hide all"> <span class="ons-btn__inner ons-js-collapsible-all-inner">Show all</span> </button>
```

### Screen reader comments: 
“The ‘show all’ accordion on the ‘example’ page does not comply with the GOV.UK Design System guidelines as the state of the accordion is not announced. Including the collapse or expand within the button would make it simpler to understand if the form label is expanded or collapsed.”

### Solution:

We recommend that the GOV Design System accordions are adopted to the office for national statistics accordions. The GOV Design System accordions are known to be easy to use.
See GOV Design System Accordions for an example.
